src/App.js

```diff
@@ -1,9 +1,67 @@
-import "./App.css";
+import CoffeeMatch from "./CoffeeMatch";
 
 const App = () => {
+  const matches = [
+    {
+      name: "aynu",
+      image: "./assets/aynu.png",
```

Comment:
Let's use the full URLs from the website, let's not upload images to the assets folder here.